apps/rps-game/src/contexts/RpsGameContext.tsx

```diff
@@ -635,7 +641,10 @@ export default function RpsGameContextProvider({ children }: Props) {
       const res = await executeScript(GET_RPS_WIN_LOSS, (arg: any, t: any) => [
         arg(playerAddress, t.Address),
         arg(nftID, t.UInt64),
-      ])
+      ]).catch(() => {
+        resetSession();
+        setIsOldSession(true);
```

Comment:
Bear with me, I'm a TypeScript noob - what happens after `setIsOldSession()` - is the user then taken through the walletless onboarding flow? Same Q for the instance below.

Reply:
setIsOldSession forces the Modal to display to the user.

Reply:
When the user clicks the "Reload!" button the page is refreshed and the user should show up as a new user.